WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -839,7 +837,7 @@ class MySiteViewModel @Inject constructor(
         mySiteSourceManager.onResume(isSiteSelected)
         isSiteSelected = false
         checkAndShowQuickStartNotice()
-        _onShowSwipeRefreshLayout.postValue(Event(mySiteDashboardPhase2FeatureConfig.isEnabled()))
+        _onShowSwipeRefreshLayout.postValue(Event(true))
```

Comment:
❓ I did brief testing by removing `_onShowSwipeRefreshLayout` live data. Refresh was working fine, Do we still need this variable?. I am not sure about the scenario where it would be needed, just pointing it out to bring it to your notice.

Reply:
Good point! Removed in a378770.